tutorials/hana-clients-choose-hana-instance/hana-clients-choose-hana-instance.md

```diff
@@ -48,13 +48,13 @@ Here are a few benefits of using SAP HANA in the cloud:
 
 * Software updates are automatically applied by SAP.
 
-* Hardware is managed by the cloud provider (e.g. AWS, Azure, GCP, or SAP).
+* Hardware is managed by a cloud provider (e.g. AWS, Azure, or GCP).
 
 * [Backups](https://help.sap.com/viewer/db19c7071e5f4101837e23f06e576495/cloud/en-US/89d71f01daca4ecaaa069d6a060167f5.html) are automatic and recovery requests are handled via [service requests](https://help.sap.com/viewer/db19c7071e5f4101837e23f06e576495/cloud/en-US/918e714867e1409da47fa01ce03ba8f3.html).  
 
 * The memory, compute and storage settings can be changed as your needs change.  
 
-* The ability is provided to expand data storage from in-memory to native storage extensions to a data lake, while providing a common access layer that enables you to have further control over performance and cost.  See also [Lower Your Data Management Costs With SAP HANA Cloud](https://saphanacloudservices.com/hana-cloud/resources/lower-your-data-management-costs-with-sap-hana-cloud/).
+* The ability is provided to expand data storage from in-memory, to native storage extensions, to a data lake, while providing a common access layer that enables you to have further control over performance and cost.  See also [Lower Your Data Management Costs With SAP HANA Cloud](https://saphanacloudservices.com/hana-cloud/resources/lower-your-data-management-costs-with-sap-hana-cloud/).
```

Comment:
Remove one space before "See also..."

Reply:
It seems that HTML ignores the additional space. So there is no need to remove it.